torch/_utils_internal.py

```diff
@@ -34,7 +34,7 @@ def resolve_library_path(path):
     return os.path.realpath(path)
 
 
-def get_source_lines_and_file(obj):
+def get_source_lines_and_file(obj, error_msg=None):
```

Comment:
Is there every an instance where we would want `error_msg` to be None instead of `torch._C.ErrorReport.call_stack()` ? Can we do that instead ?

Reply:
This is used in some non-jit places (like our serialization), so overloading it with that here might lead to weird errors (like if the call stack is built up, then someone serializes something that has an error it will print unrelated jit stuff)